main.py

```diff
@@ -44,6 +48,37 @@ def signup_data():
             cur.close()
             return render_template("chef.html")
 
+@app.route("/login", methods=['POST', 'GET'])
+def login_data():
+    msg = ""
+    if request.method == 'POST' and 'name' in request.form and 'password' in request.form:
+        name = request.form['name']
+        password = request.form['password']
+        role = request.form['role']
+
+        cursor = mysql.connection.cursor(MySQLdb.cursors.DictCursor)
+        cursor.execute(
+            "SELECT * FROM SignUp WHERE name = % s AND password = % s AND role = %s",
```

Comment:
Change the table name to something meaningful? `SignUp` is not good name to use as a table name?

Reply:
Yeah! sure subbu.

Reply:
Check this: https://anandarajpandey.com/2015/05/10/mysql-naming-coding-conventions-tips-on-mysql-database/

Reply:
The table name `users_data ` is good!

Reply:
I read the above tutorial and decide to place the table name as `users`.